standard/bridge-v1/cmd/relayer/main.go

```diff
@@ -2,169 +2,204 @@ package main
 
 import (
 	"fmt"
+	"io"
+	"net"
 	"os"
 	"os/signal"
 	"path/filepath"
-	"standard-bridge/pkg/relayer"
+	"slices"
+	"strconv"
 	"strings"
 	"syscall"
 	"time"
 
+	"standard-bridge/pkg/relayer"
+	"standard-bridge/pkg/util"
+
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
-	"github.com/rs/zerolog"
-	"github.com/rs/zerolog/log"
 	"github.com/urfave/cli/v2"
-	"gopkg.in/yaml.v2"
+	"github.com/urfave/cli/v2/altsrc"
 )
 
 const (
-	defaultHTTPPort = 8080
+	defaultHTTPPort  = 8080
+	defaultConfigDir = "~/.mev-commit-bridge"
+	defaultKeyFile   = "key"
 )
 
 var (
-	optionConfig = &cli.StringFlag{
-		Name:     "config",
-		Usage:    "path to relayer config file",
-		Required: false, // Can also set config via env var
-		EnvVars:  []string{"STANDARD_BRIDGE_RELAYER_CONFIG"},
+	portCheck = func(c *cli.Context, p int) error {
```

Comment:
`c` is unused in this func and is not directly used as an `Action` so it can be removed